Makefile

```diff
@@ -9,7 +9,14 @@ DOCKER_COMPOSE_CMD = docker-compose -f docker-compose.yml
 ifndef JENKINS_URL
   DOCKER_COMPOSE_CMD += -f docker-compose.development.yml
 endif
-TEST_CMD = $(DOCKER_COMPOSE_CMD) run publishing-e2e-tests bundle exec rspec
+
+BASE_TEST_COMMAND = $(DOCKER_COMPOSE_CMD) run publishing-e2e-tests bundle exec rspec
+
+ifdef TEST_FOCUS_PATH
+  TEST_CMD = $(BASE_TEST_COMMAND) $(TEST_FOCUS_PATH)
+else
+  TEST_CMD = $(BASE_TEST_COMMAND) --tag ~flaky
+endif
```

Comment:
I find this a little confusing. It might be simpler to use variable overriding to set this up. For example, if you have:

```make
TEST_ARGS = --tag ~flaky
TEST_CMD = $(DOCKER_COMPOSE_CMD) run publishing-e2e-tests bundle exec rspec $(TEST_ARGS)
```

You can then override the TEST_ARGS when invoking make, e.g.:

```bash
lucida:  ~/Projects/GDS/govuk/publishing-e2e-tests  |master U:1 ✗|
→ make test
echo docker-compose -f docker-compose.yml -f docker-compose.development.yml run publishing-e2e-tests bundle exec rspec --tag ~flaky
docker-compose -f docker-compose.yml -f docker-compose.development.yml run publishing-e2e-tests bundle exec rspec --tag ~flaky

lucida:  ~/Projects/GDS/govuk/publishing-e2e-tests  |master U:1 ✗|
→ make test TEST_ARGS=/foo
echo docker-compose -f docker-compose.yml -f docker-compose.development.yml run publishing-e2e-tests bundle exec rspec /foo
docker-compose -f docker-compose.yml -f docker-compose.development.yml run publishing-e2e-tests bundle exec rspec /foo
```